sandbox.ipynb

```diff
@@ -34,7 +34,7 @@
   },
```

Comment:
Do not push changes of `sandbox`. It is still under developing.

Reply:
The modification of sandbox is tiny. The version is almost the same with the version in 2.0-dev.

Reply:
Agreed. Don't push sandbox changes here. It may cause conflicts.

Reply:
Fixed.